src/web.ts

```diff
@@ -16,8 +16,8 @@ import type {
   PurchasesPromotionalOffer,
   PurchasesStoreProduct,
   ShouldPurchasePromoProductListener,
-} from '@revenuecat/purchases-typescript-internal';
-import { REFUND_REQUEST_STATUS } from '@revenuecat/purchases-typescript-internal';
+} from '@revenuecat/purchases-typescript-internal-esm';
+import { REFUND_REQUEST_STATUS } from '@revenuecat/purchases-typescript-internal-esm';
```

Comment:
can we move this to the previous import?

Reply:
Oh for some reason I didn't see these comments in github before merging :/. Will fix this though 👍